UnityProject/Assets/Scripts/UI/UI Bottom/Hands.cs

```diff
@@ -78,9 +78,26 @@ public void Use()
             //This checks which UI slot the item can be equiped too and swaps it there
             var type = Slots.GetItemType(CurrentSlot.Item);
             var masterType = Slots.GetItemMasterType(CurrentSlot.Item);
+            var itemObj = Slots.GetGameObject(CurrentSlot.Item);
 
             switch (masterType)
             {
+                case SpriteType.Food:
+                    if (itemObj.GetComponent<BaseFood>() != null)
+                    {
+                        BaseFood foodScript = itemObj.GetComponent<BaseFood>();
+                        foodScript.EatFood();
+                        Destroy(itemObj);
+
+                    }
+                    else
+                    {
+
+                        Debug.Log("Item does not have food script! Returning...");

```

Comment:
This should only get triggered on food anyway. 
So why write this unneeded check?

In development checks are great, but just cut them, or comment them out, out while done..

This is 15 rows of code, that could have been 5.